<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Eyetracker</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>res/eye.png</normaloff>res/eye.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_pupil">
      <property name="title">
       <string>Pupil</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="label_pupil_binary">
         <property name="text">
          <string>Binary Thresh</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="horizontalSlider_pupil_binary">
         <property name="minimum">
          <number>100</number>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_pupil_binary_value">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_pupil_min">
         <property name="text">
          <string>Min Size</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="horizontalSlider_pupil_min">
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>4000</number>
         </property>
         <property name="value">
          <number>99</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_pupil_min_value">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_pupil_max">
         <property name="text">
          <string>Mask Circle</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="horizontalSlider_pupil_mask_circle">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>39</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_pupil_mask_value">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_general">
      <property name="title">
       <string>General</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="label_general_blur">
         <property name="text">
          <string>Blur</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="horizontalSlider_general_blur">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_general_dilation">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_general_blur_2">
         <property name="text">
          <string>Dilation</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_general_erosion">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="horizontalSlider_general_erosion">
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_general_blur_3">
         <property name="text">
          <string>Erosion</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_general_blur_value">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="horizontalSlider_general_dilation">
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QCheckBox" name="checkBox_equalize">
         <property name="text">
          <string>Equalize Histogram</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_led">
      <property name="title">
       <string>LED</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_led_binary">
         <property name="text">
          <string>Binary Thresh</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="horizontalSlider_led_binary">
         <property name="minimum">
          <number>100</number>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_led_binary_value">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_led_min">
         <property name="text">
          <string>Min Size</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="horizontalSlider_led_min">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_led_min_value">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_led_max">
         <property name="text">
          <string>Mask Circle</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="horizontalSlider_led_mask_circle">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_led_mask_value">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="groupBox_Output">
      <property name="title">
       <string>Output</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_saveConfig">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>res/save_config.png</normaloff>res/save_config.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_pauseplay">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>res/pause.png</normaloff>res/pause.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="pushButton_process">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>res/process.png</normaloff>res/process.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_loadConfig">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>res/load_config.png</normaloff>res/load_config.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Display</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox_markup">
         <property name="text">
          <string>Image markup</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkBox_showProcessing">
         <property name="text">
          <string>Show Processing</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox_processing">
         <item>
          <property name="text">
           <string>pupil_binary</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>led_binary</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>pupil_region</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>led_region</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>eroded</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>edges</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>combined</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Details</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <widget class="QLabel" name="label_fpstext">
         <property name="text">
          <string>FPS:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_azimuth">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_azimuthtext">
         <property name="text">
          <string>Azimuth:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_fps">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="label_zenith">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_zenithtext">
         <property name="text">
          <string>Zenith:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>LED Position:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="label_ledPosition">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_pupiltext">
         <property name="text">
          <string>Pupil Area:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_pupil_area">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Pupil Position:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_pupilPosition">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>694</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuLoad">
     <property name="title">
      <string>Load</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>res/load.png</normaloff>res/load.png</iconset>
     </property>
     <addaction name="action_load_movie"/>
     <addaction name="action_load_hdf5"/>
     <addaction name="action_load_tiff"/>
     <addaction name="separator"/>
     <addaction name="action_custom_algorithm"/>
    </widget>
    <widget class="QMenu" name="menuSave_2">
     <property name="title">
      <string>Save</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>res/save.png</normaloff>res/save.png</iconset>
     </property>
     <addaction name="action_save_avi"/>
     <addaction name="action_save_hdf5"/>
     <addaction name="action_save_tiff"/>
    </widget>
    <addaction name="menuLoad"/>
    <addaction name="menuSave_2"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionVideo"/>
    <addaction name="actionPlots"/>
    <addaction name="actionModel"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="action_load_movie">
   <property name="text">
    <string>movie</string>
   </property>
  </action>
  <action name="action_load_hdf5">
   <property name="text">
    <string>hdf5</string>
   </property>
  </action>
  <action name="actionAvi">
   <property name="text">
    <string>avi</string>
   </property>
  </action>
  <action name="actionTiff_stack">
   <property name="text">
    <string>hdf5</string>
   </property>
  </action>
  <action name="actionPlots">
   <property name="text">
    <string>Plots</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="action_save_avi">
   <property name="text">
    <string>avi</string>
   </property>
  </action>
  <action name="action_save_hdf5">
   <property name="text">
    <string>hdf5</string>
   </property>
  </action>
  <action name="action_save_tiff">
   <property name="text">
    <string>tiff stack</string>
   </property>
  </action>
  <action name="action_load_tiff">
   <property name="text">
    <string>tiff stack</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionModel">
   <property name="text">
    <string>Model</string>
   </property>
  </action>
  <action name="action_custom_algorithm">
   <property name="text">
    <string>custom algorithm</string>
   </property>
  </action>
  <action name="actionVideo">
   <property name="text">
    <string>Video</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_led_binary</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_led_binary_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_led_mask_circle</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_led_mask_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>530</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_led_min</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_led_min_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>463</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_general_blur</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_general_blur_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>601</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>763</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_pupil_binary</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_pupil_binary_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>619</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>757</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_pupil_mask_circle</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_pupil_mask_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>619</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>757</x>
     <y>530</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_pupil_min</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_pupil_min_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>619</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>757</x>
     <y>463</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_general_dilation</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_general_dilation</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>671</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_general_erosion</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_general_erosion</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>671</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
